examples/quickstart/jupyter-notebooks/druidapi/druidapi/druid.py

```diff
@@ -116,6 +117,33 @@ def datasources(self) -> DatasourceClient:
         if not self.datasource_client:
             self.datasource_client = DatasourceClient(self.rest_client)
         return self.datasource_client
+    
+    def basic_security(self, authenticator, authorizer=None):
+        '''
+        Returns a client to work with a basic authorization authenticator/authorizer pair.
+        This client assumes the typical case of one authenticator and one authorizer. If
+        you have more than one, create multiple clients.
+
+        The basic security API is not proxied through the router: it must work directly with
+        the coordinator. Create an ad-hoc Druid client for your coordinator. Because you have
+        Basic security enabled, you must specify the admin user and password:
+
+        ```
+        coord = druidapi.jupyter_client('http://localhost:8081', auth=('admin', 'admin-pwd'))
+        ac = coord.basic_security('yourAuthenticator', 'yourAuthorizer')
+        ```
+
+        Parameters
+        ----------
+        authenticator: str
+            Authenticator name as set in the `druid.auth.authenticatorChain`
+            runtime property.
+
+        authorizer: str, default = authenticator
+            Authorizor name as set in the `druid.auth.authorizers` runtime property.
+            Defaults to the same as `authenticator` for simple cases.
```

Comment:
Is the default value of `authorizer` the string "authenticator", or is the default value the same value of what's assigned to `authenticator`?